function.js

```diff
@@ -1,3 +1,14 @@
+function scrollingTop() {
```

Comment:
Hello for this function, not entirely sure how setTimeout function gives the illusion of animation of scrolling to the top, when the delay is 0. But it works..